recipes-graphics/mesa/tegra-udrm-gbm_1.0.2.bb

```diff
@@ -18,4 +18,9 @@ S = "${WORKDIR}/git"
 
 inherit meson pkgconfig
 
+do_install:append:tegra234() {
```

Comment:
If you make this SoC-specific, we'd have to set the PACKAGE_ARCH to make the package SoC-specific, too.

I don't think it will hurt to add this symlink unconditionally, so just drop the `:tegra234` here.

Reply:
I will add `PACKAGE_ARCH`.
FYI, `nvidia-drm` kernel module exists only for `tegra234`

Reply:
I'd prefer it the other way.  Don't set PACKAGE_ARCH, and just create the symlink for all tegras.  The fact that only one of the two kernel modules would be active on a particular machine doesn't matter.